app/models/finances_user.rb

```diff
@@ -1,4 +1,8 @@
 class FinancesUser < ApplicationRecord
   belongs_to :user
   belongs_to :finance
+
+  scope :by_user_id, ->(id){where(user_id: id)}
+  scope :by_status_true, ->{where(status: true)}
```

Comment:
```suggestion
  scope :by_status, (status)->{where status: status}
```
rồi dùng chung đi thanh niên

Reply:
Yes, sir